README.md

```diff
@@ -17,6 +17,60 @@ Your frontend gets a beautiful, safe, and customizable login screen. Your backen
 - Full reference this library is [here](https://docs.propelauth.com/reference/frontend-apis/js)
 - Getting started guides for PropelAuth are [here](https://docs.propelauth.com/)
 
+## Installation and Usage
+
+```bash
+npm install @propelauth/javascript
+```
+
+and then you can import it:
+
+```js
+import { createClient } from "@propelauth/javascript";
+```
+
+Alternatively, you can use a CDN:
+
+```html
+<script 
```

Comment:
This version is out of date, might be easier to just make it generic and let the user fill it in.